apps/_docs/content/docs/drizzle.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Drizzle
+description: How to switch from Prisma to Drizzle.
+---
+
+next-forge uses [Prisma](https://prisma.io) as its ORM by default. However, you can easily switch to another ORM like [Drizzle](https://drizzle.dev) or [Kysely](https://kysely.dev/).
+
+We selected Prisma because as of writing this, it's the most popular ORM and has a large community and many adapters. However, Drizzle is coming up fast and has some great benefits, primarily around performance.
+
+If you want to switch to Drizzle, you have two options:
+
+1. Keep Prisma and add the Drizzle API to the Prisma client. Drizzle have a [great guide](https://orm.drizzle.team/docs/prisma) on how to do this.
+2. Go all-in and switch to Drizzle.
+
+Here, I'll assume you have a working Neon database and cover the second option.
+
+## 1. Swap out the required dependencies in `@repo/database`
+
+Uninstall the existing dependencies...
+
+```package-install
+npm uninstall @prisma/adapter-neon @prisma/client prisma --filter database
+```
+
+...and install the new ones:
+
+```package-install
+npm install drizzle-orm dotenv --filter database
+npm install -D drizzle-kit --filter database
+```
+
+## 2. Update the database connection code
+
+Delete everything in `@repo/database/index.ts` and replace it with the following:
+
+```ts title="packages/database/index.ts"
+import 'server-only';
+
+import { neon } from '@neondatabase/serverless';
+import { config } from 'dotenv';
+import { drizzle } from 'drizzle-orm/neon-http';
+
+config({ path: '.env' });
+
+const databaseUrl = process.env.DATABASE_URL;
+
+if (!databaseUrl) {
+  throw new Error('Missing DATABASE_URL environment variable.');
+}
+
+const client = neon(databaseUrl);
```

Comment:
as of our latest API improvements - you can do
```ts
import { drizzle } from "drizzle-orm/neon-http";

const db = drizzle(process.env.DATABASE_URL);
```

Reply:
Does this need to be updated? https://orm.drizzle.team/docs/tutorials/drizzle-with-neon#connect-drizzle-orm-to-your-database